test/e2e/specs/site-editor/style-variations.spec.js

```diff
@@ -14,19 +14,34 @@ test.describe( 'Global styles variations', () => {
 		await requestUtils.activateTheme(
 			'gutenberg-test-themes/style-variations'
 		);
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-global-styles'
+		);
 		await requestUtils.deleteAllTemplates( 'wp_template' );
 		await requestUtils.deleteAllTemplates( 'wp_template_part' );
+		// Delete all user global styles
+		await requestUtils.rest( {
+			method: 'DELETE',
+			path: '/wp/v2/delete-all-global-styles',
+		} );
```

Comment:
Do you think it's worth adding this as a util in request utils directly? Like the `deleteAllTemplates` util above.